x/oracle/keeper/oracle_test.go

```diff
@@ -133,10 +132,11 @@ func (suite *oracleTestSuite) TestRegisterOracleSuccess() {
 	suite.Require().Equal(suite.oracleCommissionMaxChangeRate, oracleFromKeeper.OracleCommissionMaxChangeRate)
 }
 
-func (suite *oracleTestSuite) TestRegisterOracleFailedValidateToMsgOracleRegistration() {
```

Comment:
Accordingly [this](https://github.com/medibloc/panacea-core/pull/582/files#r1059222830), deleted validation test code.